script/Ciel/Day 10/10_01_CIEL10_6.txt

```diff
@@ -1297,47 +1434,52 @@ Releasing it on me, on the third person dazedly standing around, such was the wo
 [sha:7b956eb6a50f632dcfedaed309eb4f2c4f2395ad]{
 -- Page 507, Offset 28496.
 -- ……だが、その秘策は実行すらされなかった。
-However... That secret plan could not even be set into motion.
+Except, she doesn't even get a chance to carry out the second part of her plan.
 }
 [sha:aba7f14c6a9b1ac1c865b8836b70c458f1fac491]{
 -- Page 507, Offset 28497.
 -- <女|ノエル>が黒鍵三本を放った時点で、その右肩にはシエル先輩の黒鍵が突き刺さっていたのだから。
-Indeed, the moment Noel throws her three Black Keys, her right shoulder gets transpierced by Ciel-senpai's Black Key.
+The instant those three Black Keys left her hand,
+ her right shoulder was skewered by one of Ciel-senpai's Black Keys.
```

Comment:
The causality indicator does good work here. Keep it.
>Because the instant...

Reply:
I dunno man, the causality feels pretty darn clear without it. Adding an extraneous 'because' to the start of a sentence feels like the sort of thing my high school English teacher would castrate me for.

Reply:
Yeah, but let's be real, what has your high school English teacher ever been right about? What is this institutional resentment of conjugations anyway...